.circleci/config.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           no_output_timeout: 30m
       - restore_cache:
           keys:
-            - cargo-v26b-{{ checksum "rust-toolchain" }}-{{ checksum "Cargo.toml" }}-{{ checksum "Cargo.lock" }}-{{ arch }}
+            - cargo-v27a-{{ checksum "rust-toolchain" }}-{{ checksum "Cargo.toml" }}-{{ checksum "Cargo.lock" }}-{{ arch }}
```

Comment:
lets stay at 26, just go to 26c, also this shouldn't change the groth params, or am I missing sth?

Reply:
I tried 26c, but it didn't work (missing params), so I bumped to v27 just to see if I could get that working.  The params are updated because this PR bumps some of the test sectors to have min of 2 layers (instead of 1 for the smaller test ones).

Reply:
we only need to change the full version if we switch from already published ones, because v26 is not yet published, there is no need to change that version yet